src/app/client/src/app/modules/profile/components/profile-badge/profile-badge.component.html

```diff
@@ -3,8 +3,8 @@
   <div class="header announcementCard-header">
     <span class="header cardsHeading">{{resourceService?.frmelmnts?.lbl?.certificationAward}}</span>
   </div>
-  <div *ngIf="badgeArray && badgeArray.length" class="ui items margin-top-10">
-    <div class="item" *ngFor="let badge of badgeArray">
+  <div *ngIf="badgeArray" class="ui items margin-top-10">
```

Comment:
Not as per latest change in angular-migration branch.

Expected:
<div *ngIf="badgeArray && badgeArray.length" class="ui items margin-top-10">